critiquebrainz/frontend/messages.pot

```diff
@@ -211,7 +211,7 @@ msgid "Please sign in to access this page."
 msgstr ""
 
 #: critiquebrainz/frontend/login/__init__.py:46
-msgid "You must be an administrator to view this page."
+msgid "You must be a moderator to view this page."
```

Comment:
You don't need to change the `.pot` file. See http://babel.pocoo.org/en/latest/cmdline.html. CB uses `pybabel` for compiling translations. Make the changes just in the python/html code.